src/coreclr/gc/unix/gcenv.unix.cpp

```diff
@@ -1042,13 +1079,14 @@ size_t GCToOSInterface::GetCacheSizePerLogicalCpu(bool trueSize)
 {
     static volatile size_t s_maxSize;
     static volatile size_t s_maxTrueSize;
+    bool useSysConf = GCConfig::GetGCCacheSizeFromSysConf();
 
     size_t size = trueSize ? s_maxTrueSize : s_maxSize;
     if (size != 0)
         return size;
 
     size_t maxSize, maxTrueSize;
-    maxSize = maxTrueSize = GetLogicalProcessorCacheSizeFromOS(); // Returns the size of the highest level processor cache
+    maxSize = maxTrueSize = GetLogicalProcessorCacheSizeFromOS(useSysConf); // Returns the size of the highest level processor cache
```

Comment:
do we really need to change the signature of `GetLogicalProcessorCacheSizeFromOS`? `GetCacheSizePerLogicalCpu` doesn't have any use of knowing this config value.